scripts/initialise_docker_environment.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2018 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+docker network create aether_internal
```

Comment:
`docker network create aether_internal 2>/dev/null || true` to avoid _existential_ issues :stuck_out_tongue: 

Obviously same comment with the volume command.